libs/accounts/src/lib/use-account-balance.tsx

```diff
@@ -33,9 +33,9 @@ export const useAccountBalance = (assetId?: string) => {
 
   return useMemo(
     () => ({
-      accountBalance,
-      accountDecimals,
+      accountBalance: pubKey ? accountBalance : '',
```

Comment:
why we need this? Because state isn't reset after partyId change? If so, maybe such kind of reset should be implemented instead?